src/Switch/Switch.js

```diff
@@ -54,11 +54,19 @@ export const styleSheet = createStyleSheet('Switch', (theme) => {
       height: 20,
       borderRadius: '50%',
     },
+    disabled: {
+      opacity: 0.5,
```

Comment:
I wouldn't use the opacity approach here.
What do you think about the Angular solution?

#### Disabled
![capture d ecran 2016-12-03 a 14 37 36](https://cloud.githubusercontent.com/assets/3165635/20859719/ff683212-b966-11e6-92cf-c82782b2e0d5.png)